feeluown/containers/player_control_panel.py

```diff
@@ -27,9 +27,61 @@ class SongBriefLabel(QLabel):
 
     def __init__(self, app):
         super().__init__(text=self.default_text, parent=None)
+        self.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Preferred)
         self._app = app
+        self.t = QTimer()
+        self.txt = self.raw_text = '...'
+        self.font_metrics = QFontMetrics(QApplication.font())
+        self.text_rect = self.font_metrics.boundingRect(self.raw_text)
+        self.newX = 0
```

Comment:
we'd better use  underline in variables name.

Reply:
Can we add some comments for the variable `newX` since we can't guess the meaning from the naming.

Reply:
Sorry for that, now it's more readable.